idaes/models/properties/examples/saponification_thermo.py

```diff
@@ -135,12 +136,48 @@ def define_metadata(cls, obj):
         )
 
 
+class SaponificationPropertiesScaler(CustomScalerBase):
+    UNIT_SCALING_FACTORS = {
+        # "QuantityName: (reference units, scaling factor)
+        "Pressure": (units.Pa, 1e-5),
+    }
+
+    DEFAULT_SCALING_FACTORS = {
+        "flow_vol": 1e2,
+        "conc_mol_comp": 1e-2,
+    }
+
+    def variable_scaling_routine(
+        self, model, overwrite: bool = False, submodel_scalers: dict = None
+    ):
+        self.scale_variable_by_default(model.flow_vol, overwrite=overwrite)
```

Comment:
Is there a way for the user to provide a default value, rather than just using whatever happens to be the default default value?

Reply:
Yes, although it might not be the best way to do it. The default values are in a class attribute (dict) which users can interact with. That said, that is probably not the most obvious API for it (we could have a method to update the defaults).